editor/editor_themes.h

```diff
@@ -53,6 +53,22 @@ class EditorColorMap {
 	static HashSet<StringName> &get_color_conversion_exceptions() { return color_conversion_exceptions; };
 };
 
+class EditorTheme : public Theme {
+	GDCLASS(EditorTheme, Theme);
+
+	static Vector<StringName> editor_theme_types;
```

Comment:
Why is this static if it's not used in a static context? Also there is only one EditorTheme.

Reply:
Well, if for some reason there is more than one EditorTheme instanced at the same time (even if transiently), they don't need to duplicate this list which is tied to the editor as a whole.

Basically, I made it static because it's not tied to any instance.